src/main/java/org/linksharing/server/http/ApiRestController.java

```diff
@@ -67,7 +67,16 @@ ResponseEntity<UserProfileDetails> updateProfileDetails(Principal user, @Request
 
     @GetMapping("/profile/picture")
     ResponseEntity<?> getProfilePicture(Principal user) throws FileNotFoundException {
-        File img = new File("../image.jpg");
+
+        String imageUrl;
+
+        try {
+            imageUrl = profileRepository.findByEmail(user.getName()).getImageUrl();
+        } catch (NullPointerException e) {
```

Comment:
Usually, we don't catch NPE. The better way is check is something is `null` or not.

Reply:
Ok. Fixed.